metrics/plugins/json.lua

```diff
@@ -6,6 +6,11 @@ local function finite(value)
     if type(value) == "string" then
         value = tonumber(value)
         if value == nil then return nil end
+    elseif type(value) == "cdata" then
+        value = tostring(value)
+        -- Luajit cdata type inserts some postfix in the end of the number after tostring() operation
+        value = value:gsub("U*LL", "")
+        return tonumber(value)
```

Comment:
How about numbers out of the range [-2^53; 2^53] range, where double (lua number) types cannot represent all integral numbers? There is `number64()` function. BTW, it returns just `number` if the value is small enough.

(I'm not in context here, but, it seems, this part of the code may be just `number64()` call.)

Reply:
Oh, thank you. I've based my solution on https://github.com/tarantool/metrics/pull/23 , but I think it is worth to use more simpler one

Reply:
Reworked

Reply:
I still don't understand why `tonumber()` is there. It'll break some numbers outside of the [-2^53; 2^53] range:

```lua
tarantool> tonumber(9007199254740993ULL)
---
- 9007199254740992
...

tarantool> tonumber64(9007199254740993ULL)
---
- 9007199254740993
...
```

Reply:
Well, dodging precision loss is impossible until we support encoding cdata to numbers in json package. Metric value as string is not supported by Telegraf/InfluxDB, who is the main (if not the only one) consumer of json metrics.

Reply:
Seems that I am wrong about cdata transform.

Reply:
Okay.